pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: wolt_responsive_layout_grid
-description: Build responsive grid layouts with ease.
+description: Wolt Responsive Layout Grid library helps you to build responsive grid layouts with ease.
```

Comment:
Are we sure this is long enough? How about this:
Wolt Responsive Layout Grid library helps you to create dynamic, adaptive grid layouts that seamlessly adjust to various screen sizes.

Reply:
Min 60 chars is good enough but your description is better, let me update 👍🏽

Reply:
addressed in d20472a